src/v2/guide/comparison.md

```diff
@@ -120,7 +120,7 @@ Since you don't need to know about JSX, ES2015, or build systems to get started
 
 ### Native Rendering
 
-React Native enables you to write native-rendered apps for iOS and Android using the same React component model. This is great in that as a developer, you can apply your knowledge of a framework across multiple platforms. On this front, Vue has an official collaboration with [Weex](https://alibaba.github.io/weex/), a cross-platform UI framework developed by Alibaba Group, which uses Vue as its JavaScript framework runtime. This means with Weex, you can use the same Vue component syntax to author components that can not only be rendered in the browser, but also natively on iOS and Android!
+React Native enables you to write native-rendered apps for iOS and Android using the same React component model. This is great in that as a developer, you can apply your knowledge of a framework across multiple platforms. On this front, Vue has an official collaboration with [Weex](https://weex.apache.org/), a cross-platform UI framework driven by Alibaba Group and being incubated to Apache Software Foundation (ASF), which uses Vue as its JavaScript framework runtime. This means with Weex, you can use the same Vue component syntax to author components that can not only be rendered in the browser, but also natively on iOS and Android!
```

Comment:
"... which uses Vue as one of its ..."

I think adding "one of" will be more accurate.

Reply:
Updated. Is that OK?